tests/main.js

```diff
@@ -406,10 +410,26 @@ testBundlers('Can reduce parallelism', [ESBUILD, ESBUILD_ZISI, DEFAULT], async (
 
 testBundlers('Can use zipFunction()', [ESBUILD, ESBUILD_ZISI, DEFAULT], async (bundler, t) => {
   const { path: tmpDir } = await getTmpDir({ prefix: 'zip-it-test' })
-  const { runtime } = await zipFunction(`${FIXTURES_DIR}/simple/function.js`, tmpDir, {
+  const result = await zipFunction(`${FIXTURES_DIR}/simple/function.js`, tmpDir, {
     config: { '*': { nodeBundler: bundler } },
   })
-  t.is(runtime, 'js')
+  const getOutBundler = () => {
+    switch (bundler) {
+      case ESBUILD_ZISI:
+        return ESBUILD
+
+      case DEFAULT:
+        return JS_BUNDLER_ZISI
+
+      default:
+        return bundler
+    }
+  }
+  const outBundler = getOutBundler()
```

Comment:
Would an object work in this case?

```js
const outBundlers = { [ESBUILD_ZISI]: ESBUILD, [DEFAULT]: JS_BUNDLER_ZISI }
const outBundler = outBundlers[bundler] || bundler
```

Reply:
Sure! Done in https://github.com/netlify/zip-it-and-ship-it/pull/410/commits/2826d4bb1632130f7e458c8857346fd0882b4321.